guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,411 @@ Illustration sources:
+   width = 250
+   height = 541
+%}
 
-To fully restore a wallet with funds on the [Lightning network]({{ '/guide/getting-started/technology-primer/#the-lightning-payment-network' | relative_url }}), the state and history of the payment channels are needed in addition to the recovery phrase. As there are no standards for this yet, wallet applications need to have their own format for backing this up, and subsequently restoring. Due to the amount of information, a QR code of the encrypted data, like in the previous example, can be appropriate.
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   layout = "shadow"
+   caption = "App begins to restore, pulling the channels backup automatically from the cloud."
+   alt-text = "Screen showing an animation as the wallet is being restored"
+   width = 250
+   height = 541
+%}
 
-{% include tip/recommendation.html %}
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   layout = "shadow"
+   caption = "Wallet is finished being restored."
+   alt-text = "Screen showing that the wallet has been restored."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   layout = "shadow"
+   caption = "The wallet has been successfully restored."
+   alt-text = "Screen showing a home screen with funds"
+   width = 250
+   height = 541
+%}
+
+</div>
 
-Be clear to users what you support and what wallet specifications they can expect to restore in your application. Most bitcoin products should support restoring from a recovery phrase at a minimum. Capability to restore an HD wallet implemented according to BIP32, 39, 43, 44, 49, 84 and 380 would be a good starting point.
+{% include tip/open.html color="green" icon="check" label="Use autocomplete" %}
+
+This wallet makes the recovery process easier for the user by autocompleting the words of their recovery phrase as they type. Since recovery phrases are comprised of words from a predefined dictionary of 2,048 words, the app is able usually able to autocomplete or detect a typo with only 3 or 4 letters typed.
```

Comment:
```suggestion
This wallet makes the recovery process easier for the user by autocompleting the words of their recovery phrase as they type. Since recovery phrases are comprised of words from a predefined dictionary of 2,048 words, the app is usually able to autocomplete or detect a typo with only 3 or 4 letters typed.
```